webgpu.yml

```
@@ -304,38 +309,34 @@ enums:
     doc: |
       TODO
     entries:
+      - null
       - name: success
-        value: 0x0001
         doc: |
           TODO
       - name: instance_dropped
-        value: 0x0002
         doc: |
           TODO
       - name: validation_error
-        value: 0x0003
         doc: |
           TODO
       - name: internal_error
-        value: 0x0004
         doc: |
           TODO
       - name: device_lost
-        value: 0x0005
         doc: |
           TODO
       - name: device_destroyed
-        value: 0x0006
         doc: |
           TODO
       - name: unknown
-        value: 0x0007
         doc: |
           TODO
   - name: cull_mode
     doc: |
       TODO
     entries:
+      - name: undefined
+        doc: No value. See @ref SentinelValues.
```

```suggestion
        doc: The absence of a value. See @ref SentinelValues.
```

I got mildly confused when reading the C header as I though the block comment meant there should not have been an enum value associated with `0x000000`